roles/openshiftpostdeployment/tasks/main.yml

```diff
@@ -30,6 +30,18 @@
     - openshift
   when: installLogging == True
 
+- name: Add default tolerations to infrastructure namespaces
+  command: /usr/local/bin/oc annotate ns {{ item }} scheduler.alpha.kubernetes.io/defaultTolerations='[{"key":"infra","value":"only","effect":"NoSchedule","operator":"Equal"}]'
+  with_items:
```

Comment:
Can we enumerate this list from the environment first? Does it have to be statically coded?